kotlin-native/backend.native/tests/build.gradle

```diff
@@ -1103,6 +1103,11 @@ if (PlatformInfo.isAppleTarget(project)) {
         UtilsKt.dependsOnPlatformLibs(it)
     }
 
+    standaloneTest("interop_objc_kt65260") {
+        source = "interop/objc/kt65260.kt"
+        UtilsKt.dependsOnPlatformLibs(it)
+    }
```

Comment:
This file is the legacy test infrastructure. We are currently migrating all tests from here.
Please add the new test there instead: https://github.com/JetBrains/kotlin/tree/master/native/native.tests/testData/codegen/cinterop/objc
After creating a file, you will need to run `generateTests` Gradle task (or `Generate Tests` run configuration in IDEA). This will generate JUnit tests for the new testdata file.

Reply:
Tests have been migrated.